### ISSUE TITLE
[Feature request] Support global data properties stored in diagram

### ISSUE THREAD

**Issue by alexanderwiller** [jgraph/drawio#549]
It seems like it is only possible to add persistent data properties (via "Edit Data...") that are page-local.

I haven't found any way to set global data properties persistently, meaning that they are not passed as URL parameters, but stored inside the diagram.

A use case for this would be to store author, revision or creation date of a document and be able to change every occurence in one step.

Basically, the same dialog as in "Edit Data..." would suffice, named like "Edit Global Data..." or similar and maybe accessible via the File menu, not via the Edit menu.
